Alcatraz/Alcatraz.m

```diff
@@ -52,7 +52,9 @@ + (NSString *)localizedStringForKey:(NSString *)key {
 - (id)initWithBundle:(NSBundle *)plugin {
     if (self = [super init]) {
         self.bundle = plugin;
-        [self createMenuItem];
+        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Mind using `NSOperationQueue` here?
It's just a convention we kept through the project.
```objc
[[NSOperationQueue mainQueue] addOperationWithBlock:^{
    [self creareMenuItem];
}];
```